bootstrapper/internal/kubernetes/kubernetes.go

```diff
@@ -218,30 +237,34 @@ func (k *KubeWrapper) InitCluster(
 		return nil, fmt.Errorf("setting up extraVals: %w", err)
 	}
 
+	log.Infof("Installing Constellation microservices")
 	if err = k.helmClient.InstallConstellationServices(ctx, helmReleases.ConstellationServices, extraVals); err != nil {
 		return nil, fmt.Errorf("installing constellation-services: %w", err)
 	}
 
+	log.Infof("Setting up internal-config ConfigMap")
 	if err := k.setupInternalConfigMap(ctx, strconv.FormatBool(azureCVM)); err != nil {
 		return nil, fmt.Errorf("failed to setup internal ConfigMap: %w", err)
 	}
 
 	// cert-manager is necessary for our operator deployments.
+	log.Infof("Installing cert-manager")
+	timeToStartWaiting = time.Now()
 	if err = k.helmClient.InstallCertManager(ctx, helmReleases.CertManager); err != nil {
 		return nil, fmt.Errorf("installing cert-manager: %w", err)
 	}
-
+	timeUntilFinishedWaiting = time.Since(timeToStartWaiting)
+	log.With(zap.Int("duration", int(timeUntilFinishedWaiting.Seconds()))).Infof("cert-manager installation finished")
```

Comment:
```suggestion
	log.With(zap.Duration("duration", timeUntilFinishedWaiting)).Infof("cert-manager installation finished")
```

Reply:
Changed to zap.Duration and also made it log for every Helm install (outside the retry loop so we get the total duration of a _successful_ install).

Wasn't sure if it makes sense to log duration for non-successful installations.